fedmsg_meta_fedora_infrastructure/bodhi.py

```diff
@@ -121,12 +129,31 @@ def subtitle(self, msg, **config):
 
         elif 'bodhi.update.complete.' in msg['topic']:
             author = msg['msg']['update']['submitter']
+            if isinstance(author, dict):
+                author = author['name']
             package = msg['msg']['update']['title']
             status = msg['msg']['update']['status']
             tmpl = self._(
                 "{author}'s {package} bodhi update completed push to {status}"
             )
             return tmpl.format(author=author, package=package, status=status)
+        elif 'bodhi.update.eject' in msg['topic']:
```

Comment:
Yeah, ``eject``.  :)

The bodhi2 mash starts with a bunch of updates and as it goes along it figures out that updates X and Y are no good and so it "ejects" them.